lib/ansible/cli/playbook.py

```diff
@@ -76,19 +78,21 @@ def run(self):
 
         # initial error check, to make sure all specified playbooks are accessible
         # before we start running anything through the playbook executor
+
+        playbook_dirs = []
```

Comment:
To make this clear, along with the next comment, we should probably make this `b_playbook_dirs`